backend/chunker/chunker.go

```diff
@@ -1465,6 +1465,20 @@ func (f *Fs) Purge(ctx context.Context, dir string) error {
 	return do(ctx, dir)
 }
 
+// Resume checks whether the (remote, ID) pair is valid and returns
+// the point the file should be resumed from or an error.
+//
+// This command will chain to `resume` from wrapped remote.
+// Can later be replaced by an implementation of `resume` for chunker
+//
+func (f *Fs) Resume(ctx context.Context, remote string, ID string) (Pos int64, err error) {
+	do := f.base.Features().Resume
+	if do == nil {
+		return 0, fs.ErrorCantResume
+	}
+	return do(ctx, remote, ID)
```

Comment:
@mcalman
Please also add `UnimplementableFsMethods: []string{..., "Resume"},` to the `chunker_test.go`, and we can mark this bit as resolved

Reply:
I think you can completely remove the `func Resume()` stub from `chunker.go`. Let's address it in another PR.
Testing resume interface with chunker is not a low hanging fruit, though it looks tempting.
I think `backend/local` is a better candidate for that.

Reply:
@mcalman
I think you can completely remove the `func Resume()` stub from `chunker.go`.
Testing the resume interface with chunker is not as simple as it may seem, though tempting. Let's address it in another PR.
I think `backend/local` is a better candidate for that.
Please also add `UnimplementableFsMethods: []string{..., "Resume"},` to the `chunker_test.go` so we can mark this thread as resolved.

Reply:
@mcalman
I think you can completely remove the `func Resume()` stub from `chunker.go`.
Testing the resume interface with chunker is not as simple as it may seem, though tempting. Let's address it elsewhere.
I think `backend/local` is the best candidate for unit tests here.
Please also add `UnimplementableFsMethods: []string{..., "Resume"},` to the `chunker_test.go` so we can mark this thread as resolved.